### PR TITLE
fix: reduce padding on the pfe jump link

### PR DESCRIPTION
**Pull request by alazzara**
## <Pull request description>

Updating padding for jump-links.
- https://rally1.rallydev.com/#/270861059696d/teamboard?detail=%2Fuserstory%2F505884023220&view=113b0e72-0781-41c7-b6d6-e161d3349c72
- https://xd.adobe.com/view/1a085703-7753-4fab-9fcf-d3e9683232b4-4f5b/
- https://github.com/patternfly/patternfly-elements/issues/1274
- Update variables to new standard with fallbacks in pfe-jump-links as well as pfe-accordion in order to get the override to work in jump-links for accordion.

### Preview 

Link(s) to demo page(s) where this element can be viewed:

- [pfe-accordion demo](https://deploy-preview-1482--happy-galileo-ea79c4.netlify.app/elements/pfe-accordion/demo/)
- [pfe-jump-links demo](https://deploy-preview-1482--happy-galileo-ea79c4.netlify.app/elements/pfe-jump-links/demo/)

### What has changed and why

- pfe-jump-links.scss has been updated with new padding vars and values.
- pfe-accordion.scss has been updated with new vars so they can be overridden in jump-links.

### Testing instructions

1. Open the tabs demo page and you should see the following changes in the XD file 
2. Jump links panel should have no background color and no padding
3. Jump links panel accordion wrapped should have no padding

Sandbox page:
- https://web-dev-drupal-anbartle.dev.a1.vary.redhat.com/en/demo-web-components

Demo pages:
- https://deploy-preview-1482--happy-galileo-ea79c4.netlify.app/elements/pfe-jump-links/demo
- https://deploy-preview-1482--happy-galileo-ea79c4.netlify.app/elements/pfe-accordion/demo/

XD files:
- https://xd.adobe.com/view/793e2f7d-a1e4-414e-a5da-26ab0d6013cd-2c6d/
- https://xd.adobe.com/view/1a085703-7753-4fab-9fcf-d3e9683232b4-4f5b/

Test style overrides via the `inspector-stylesheet`
paste in

for` jump-links`
```
:root {
    --pfe-accordion__panel-container--Padding: 100px;
    --pfe-accordion__panel--Padding: 100px;
    --pfe-jump-links--BackgroundColor: pink;
}
```
For `/elements/pfe-accordion/demo/`
```
:root {
    --pfe-accordion__panel-container--Padding: 100px;
    --pfe-accordion__panel--Padding: 100px;
}
```

Make sure overrides work and default styles are as expected from mockups.

#### Browser requirements

Your component should work in all of the following environments:

- [x] Latest 2 versions of Edge
- [ ] Internet Explorer 11 (should be useable, not pixel perfect)
- [ ] Latest 2 versions of Firefox (one on Mac OS, one of Windows OS)
- [ ] Firefox 68 (or latest version for Red Hat Enterprise Linux distribution)
- [x] Latest 2 versions of Chrome (one on Mac OS, one of Windows OS)
- [ ] Latest 2 versions of Safari
- [ ] Android mobile device (such as the Galaxy S9)
- [x] Apple mobile device (such as the iPhone X)
- [ ] Apple tablet device (such as the iPhone Pro)

### Ready-for-merge Checklist

- [x] Expected files: all files in this pull request are related to one request or issue (no stragglers or scope-creep).
- [ ] Tests have been updated to cover these changes.
- [ ] Browser testing passed.
- [ ] Repository compiles and tests pass.
- [x] Changelog updated.
- [ ] Documentation (README.md, WHY.md, etc.) updated or added.
- [ ] Link to the demo recording: []()
- [ ] Approved by designer.

### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.

**Be sure to share your updates with the [patternfly-elements-contribute@redhat.com](mailto:patternfly-elements-contribute@redhat.com) mailing list!**

### CODE REVIEW

**Comment by starryeyez024** [comment]
Looks like something went a little wonky with broadcast variables 

image

**Comment by castastrophe** [comment]
> Looks like something went a little wonky with broadcast variables 
> 
> 
> 
> image
> 
> 

Can you add a few environment details in case they're needed for debugging?

**Comment by starryeyez024** [comment]
The screenshot is from the netlify page (I added the links to the PR
description).

**Comment by alazzara** [comment]
Thanks for checking/letting me know @starryeyez024 ! I will take a look at this.

File: elements/pfe-accordion/src/pfe-accordion-panel.scss [hunk 1]
```diff
@@ -19,10 +19,9 @@
 .container {
   @include pfe-clearfix;
   @include pfe-panel-container;
-  --pfe-accordion__panel-container--Padding: 0 calc(#{pfe-local(Padding, $region: base)} * 3) #{pfe-local(Padding, $region: base)} calc(#{pfe-local(Padding, $region: base)} * 1.5);
-
+  padding: pfe-local(Padding, $region: panel-container, $fallback: 0 pfe-local(Padding, $region: panel, $fallback: calc(#{pfe-local(Padding, $region: base)} * 3)) pfe-local(Padding, $region: base) pfe-local(Padding, $region: panel, $fallback: calc(#{pfe-local(Padding, $region: base)} * 1.5)));
   :host([disclosure="true"]) & {
-    --pfe-accordion__panel-container--Padding: 0 calc(#{pfe-local(Padding, $region: base)} * 1.5);
```
Comment (starryeyez024) [inline elements/pfe-accordion/src/pfe-accordion-panel.scss:25]: Note, this PR is already solving for the padding-bottom issue on the panel:

https://github.com/patternfly/patternfly-elements/pull/1443/files

Comment (castastrophe) [reply elements/pfe-accordion/src/pfe-accordion-panel.scss:25]: Ah yes except it's doing it by updating the variable and Adam's PR is doing it by keep the variable empty.  I think this is the preferred approach right?

Comment (alazzara) [reply elements/pfe-accordion/src/pfe-accordion-panel.scss:25]: +1^

**Comment by alazzara** [comment]
> Looks like something went a little wonky with broadcast variables
> 
> image

@starryeyez024 Looks like this issue is on master.

**Comment by castastrophe** [comment]
`Looks like this issue is on master.` < yeah dang, some kind of race condition with context setting.  https://github.com/patternfly/patternfly-elements/issues/1486

**Comment by starryeyez024** [comment]
@alazzara  we may need to take a second look at the mobile version , the [mockups in the Red Hat Digital Design Kit](https://xd.adobe.com/spec/3e62e93c-8338-4f31-5040-3b81f0ed5c71-4853/screen/905302c9-f428-4d06-b741-9ed83c0f0223/) look different than the [demo page](https://deploy-preview-1482--happy-galileo-ea79c4.netlify.app/elements/pfe-jump-links/demo/).

**Review by starryeyez024 — APPROVED** [review]
Lofty Goals Take Might

**Pull request merged**